dev/release.sh

```diff
@@ -319,10 +319,12 @@ echo "== Configuring OpenSSL for update and release.  This may take a bit of tim
 
 ./Configure cc >&42
 
-$VERBOSE "== Checking source file updates"
+$VERBOSE "== Checking source file updates and fips checksums"
 
 make update >&42
 
+make update-fips-checksums >&42
```

Comment:
So the plan here is that we have the checksums updated only for each release..
Is there a way of displaying all the differences between master and the release checksums currently? Or is this a manual process?

Reply:
There is a `diff-fips-checksums` Makefile target